packages/itwinui-css/src/global.scss

```diff
@@ -23,7 +23,27 @@
 // box-sizing, text selection and scrollbar styling on iui-root and all elements under it
 :where(.iui-root, .iui-root *) {
   @include mixins.iui-text-selection;
-  @include mixins.iui-scrollbar;
+  scrollbar-color: hsl(var(--iui-color-foreground-hsl) / var(--iui-opacity-4)) transparent;
```

Comment:
I'd say keep the scrollbar stuff in a local mixin at the bottom at the file. It should not be used anywhere else in the repo, but it will help with readability.

Reply:
Okay, will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1917/commits/621dd3433bcf61c53e8150c0d131abe3522c334d)